src/helm/benchmark/scenarios/legal_summarization_scenario.py

```diff
@@ -0,0 +1,194 @@
+from pathlib import Path
+
+from typing import List, Optional, Any
+
+import datasets
+from datasets import load_dataset
+
+from .scenario import Scenario, Instance, Reference, TRAIN_SPLIT, VALID_SPLIT, TEST_SPLIT, CORRECT_TAG, Input, Output
+
+_ALL_LANGUAGES = {
```

Comment:
Are these actually used?

Reply:
Currently no for simplicity. But this would also be an easy task to add more multilinguality